diesel/src/macros/insertable.rs

```diff
@@ -344,6 +344,14 @@ mod tests {
         }
     }
 
+    test_struct_definition! {
```

Comment:
There is no impl for &String: From<&str>, so this needs some more explicit conversion.